tests/common/include/tests/common/tools_single_type_navigator.inl

```diff
@@ -149,7 +149,7 @@ TEST(ALGEBRA_PLUGIN, single_type_navigator) {
     // be a surface
     ASSERT_EQ(state.next()->index, 128u);
     ASSERT_EQ(state.trust_level(),
-              toy_navigator::navigation_trust_level::e_high_trust);
+              toy_navigator::navigation_trust_level::e_full_trust);
```

Comment:
Same here as well. What is the motivation for changing the trust level from navigator side?